code/worker.sh

```diff
@@ -533,7 +535,7 @@ echo "==========================================="
 
 # If this a PR, then just check whether the files have generated
 # See https://github.com/AppImage/appimage.github.io/issues/476 for more information
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then 
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
```suggestion
if [ "$github.event.pull_request" != "false" ]; then
```
Not 100% sure about this.

Reply:
I think `"$github.event.pull_request"` is only valid in the .yml. Though according to https://stackoverflow.com/questions/61886993/in-github-actions-how-to-get-the-type-of-a-trigger-event-as-a-variable , `"${{ github.event.pull_request }}"` should work.

According to the GitHub docs, `GITHUB_HEAD_REF` only gets set for pull requests. But I should have used `-v` not '-z'.

Anyway, I've changed that to use `"${{ github.event.pull_request }}"

Reply:
> Anyway, I've changed that to use `"${{ github.event.pull_request }}"`

[Error, bad substitution](https://github.com/andy5995/appimage.github.io/runs/3001562669?check_suite_focus=true#step:4:1398)

I've changed it back to `GITHUB_HEAD_REF`

Reply:
You should be able to do something like:
test.yml
```bash
if [[ "${{ github.event.pull_request }}" = true ]]; then
          echo "IS_PULLREQUEST" >> $GITHUB_ENV
fi
```
and then

worker.sh
```bash
if [ $IS_PULLREQUEST != "false" ]; then
```

Reply:
> You should be able to do something like:
> test.yml
> 
> ```shell
> if [[ "${{ github.event.pull_request }}" = true ]]; then
>           echo "IS_PULLREQUEST=1" >> $GITHUB_ENV
> fi
> ```

I think I'm [doing it correctly](
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#example-setting-an-environment-variable), but [I don't see that it's getting set to anything](https://github.com/AppImage/appimage.github.io/pull/2560/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R9).

Reply:
Yeah I would think that should work.
The code I suggested is from a working example: https://github.com/vircadia/vircadia/blob/master/.github/workflows/pr_build.yml

Maybe they had a reason to do it the way they did, but I would have too dig far too deep into the archives to justify looking it up.

Reply:
Just realized that we set the `env` before the actual job in our actions: https://github.com/vircadia/vircadia/blob/master/.github/workflows/pr_build.yml
You put the `env` "inside" the job. Maybe try moving it accordingly?